packages/gatsby-rehype-inline-images/src/gatsby-theme-try-ghost/styles/fluid.css

```diff
@@ -3,16 +3,14 @@
 /* Fluid inline images
 /* ---------------------------------------------------------- */
 
-html.casper .post-content figure {
+html.casper .post-content {
+    display: flex!important;
```

Comment:
I think we can remove !important if we remove this adjustment https://github.com/styxlab/gatsby-theme-try-ghost/blob/master/packages/gatsby-theme-ghost-toc/src/gatsby-theme-try-ghost/styles/toc.css#L23